iOS/src/toga_iOS/libs/uikit.py

```diff
@@ -452,6 +452,16 @@ class UIViewContentMode(Enum):
     BottomRight = 12
 
 
+# UIViewAutoresizing
+UIViewAutoresizingNone = 0
+UIViewAutoresizingFlexibleLeftMargin = 1 << 0
+UIViewAutoresizingFlexibleWidth = 1 << 1
+UIViewAutoresizingFlexibleRightMargin = 1 << 2
+UIViewAutoresizingFlexibleTopMargin = 1 << 3
+UIViewAutoresizingFlexibleHeight = 1 << 4
+UIViewAutoresizingFlexibleBottomMargin = 1 << 5
```

Comment:
We've been leaning towards using Enums (e.g., UIViewContentMode above), rather than constants. There are some historical examples that use constants; these generally predate the introduction of Enum support in Rubicon.

Reply:
I used intEnum rather than Enum so that the `|` operator would work without calling `.value`.